autocomplete_light/fields.py

```diff
@@ -88,13 +85,24 @@ class MultipleChoiceField(ChoiceField, forms.MultipleChoiceField):
 class ModelChoiceField(FieldBase, forms.ModelChoiceField):
     widget = ChoiceWidget
 
+    def __init__(self, autocomplete=None, registry=None, widget=None,
+            widget_js_attributes=None, autocomplete_js_attributes=None,
+            extra_context=None, *args, **kwargs):
+
+        super(ModelChoiceField, self).__init__(autocomplete, registry, widget,
+                widget_js_attributes, autocomplete_js_attributes,
+                extra_context, *args, **kwargs)
+
+        # Store ourself in the autocomplete, for AutocompleteModel.choices.
+        self.autocomplete.model_field = self
```

Comment:
While I understand how convenient it is to have the model field instance in the autocomplete, it seems like we're coupling Autocompletes to Fields by doing that. I'm not firmly against it, but in that case shouldn't we find a more global way, that would support say django.forms.ModelChoiceField.queryset ?

Meanwhile, should we consider setting self.autocomplete.choices here directly ?